crypto/crypto.ts

```diff
@@ -33,7 +36,7 @@
  * - `BLAKE2B-256`
  * - `BLAKE2B-384`
  * - `BLAKE2S`
- * - `BLAKE3`
+ * - `BLAKE3` (length-adjustable)
```

Comment:
What's the difference of `(length-adjustable)` and `(length-entendable)`?

Reply:
> "length-extendable" refers to a specific vulnerability in some hash functions, while "length-adjustable" is not a standard term in cryptography but could refer to the general capability of hash functions to process inputs of different lengths.

-- ChatGPT